File: C99/1182.c
#include <stdio.h>
#define vertical 12
#define horizontal 12
int main() {
    int coluna, i, j, contM;
    float M[vertical][horizontal], resultado = 0;
    char escolha[1];
    scanf("%d %c", &coluna, &escolha[0]);
    for (i = 0; i < vertical; i++)
        for (j = 0; j < horizontal; j++) {
            scanf("%f", &M[i][j]);
            if (j == coluna)
                resultado += M[i][j];
        }
    if (escolha[0] == 'M' || escolha [0] == 'm')
        resultado /= vertical;
    printf("%.1f\n", resultado);
    return 0;
}

File: Python/1182.py
def bee1182():
    horizontal = 12
    vertical = 12
    coluna = int(input())
    operacao = input().upper()
    resultado = 0
    M = [[0 for i in range(horizontal)] for i in range(vertical)]
    for i in range(vertical):
        for j in range(horizontal):
            M[i][j] = float(input())
            if j == coluna:
                resultado += M[i][j]
    if operacao == 'M':
        resultado /= vertical
    print("{:.1f}".format(resultado))
bee1182()
